docs/testnet-deployments.md

```diff
@@ -495,3 +495,28 @@
 | Collateral Name | Collateral Address | TroveManager Address | Oracle Address |
 | --------------- | ------------------ | -------------------- | -------------- |
 | -               | -                  | -                    | -              |
+
+## Optimism Sepolia Testnet
```

Comment:
Add this testnet to the navigation section of this document. Just scroll up and you will see how to do that.